docs/rdiff-backup.1

```diff
@@ -72,6 +72,13 @@ using the
 .B \-\-remove-older-than
 parameter.
 .TP
+.BI "\-\-api-version " api
+Sets the API version to the given integer between minimum and maximum versions
```

Comment:
Would rephrase this to : "Force an older API version to be used instead of the default."

Reply:
No, the default is currently the oldest one, mainly based on our discussion that you want stability. This said, it might change in the future.